EVALUATION.md

```diff
@@ -0,0 +1,65 @@
+The evaluation process is done through the evaluate function that is located in the evaluation.py script (https://github.com/lovodkin93/ucca/blob/master/ucca/evaluation.py).
+A wrapping script of the evaluation.py script is evaluate_standard.py (https://github.com/lovodkin93/ucca/blob/master/scripts/evaluate_standard.py). For more details on how the scripts receives its argument, please write evaluate_standard --help in the prompt.
+The evaluation process compares the gold-standard annotation of a specific passage, with the calculated annotation of that same passage, which is calculated by our model.
```

Comment:
Avoid referring to "out model" since there is no "model" in this context. The evaluation is a general way to compare annotations of a passage, where one annotation _might_ be the output of a parser.